### PR TITLE
fix(pipeline stages): expand canary analysis description.

### PR DESCRIPTION
**Pull request by avh121**

### CODE REVIEW

File: reference/pipeline/stages/index.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ the default behavior and re-bake your image each time the pipeline runs, select
 
 ### Canary Analysis
 Run [automated canary analysis](/guides/user/canary/) against the deployment
-before fully deploying.
+before fully deploying. Note that this stage _only_ handles analysis; you're
+responsible for provisioning and cleaning up your canary instances elsewhere.
```
Comment (dorbin) [inline reference/pipeline/stages/index.md]: Does elsewhere mean in another (downstream) stage?

File: reference/pipeline/stages/index.md [hunk 2]
```diff
@@ -37,7 +37,10 @@ the default behavior and re-bake your image each time the pipeline runs, select
...
+For a step by step explanation of how to set up a canary analysis stage see the
```
Comment (dorbin) [inline reference/pipeline/stages/index.md]: step-by-step

**Review by dorbin — COMMENTED** [review]
Couple of inline comments. 
Plus: it might be good to get the name Kayenta in there somehow. This is partly because all the press for this feature has treated Kayenta as a product, practically, rather than just a microservice that supports ACA.

File: reference/pipeline/stages/index.md (hunk 1 above)
Comment (duftler) [reply reference/pipeline/stages/index.md]: Could drop elsewhere and add another sentence with something like: "This is typically done within the same pipeline via stages surrounding the Canary Analysis stage."

Comment (avh121) [reply reference/pipeline/stages/index.md]: Yeah, Matt's wording is much better -- updated it to that.

**Review by dorbin — APPROVED** [review]
:+1:

**Pull request merged**